src/data/events.json

```diff
@@ -86,5 +86,16 @@
     "location": "Chennai",
     "communityName": "Women Tech Makers (WTM) Chennai",
     "communityLogo": "https://i.ibb.co/ymLgFRk3/images.png"
-  }
+  },
+  {
+  "eventName": "MongoDB User Group Chennai: April Meetup - The AI Awakens",
+  "eventDescription": "Join us for an exciting in-person meetup where we explore the power of AI and MongoDB! Whether you're a developer, data enthusiast, or just curious about the intersection of AI and databases, this is the perfect place to learn, network, and grow.",
+  "eventDate": "2024-04-19",
+  "eventTime": "09:30",
+  "eventVenue": "IBM, 5th Floor, Unit 3, Pinnacle Building, Ascendas IT Park, CSIR Road, Tharamani, Chennai - 600113",
+  "eventLink": "https://www.mongodb.com/community/forums/t/mongodb-user-group-chennai-april-meetup-the-ai-awakens/316466",
+  "location": "Chennai",
+  "communityName": "MongoDB User Group Chennai",
+  "communityLogo": "https://www.mongodb.com/community/forums/uploads/default/original/3X/4/9/49957b7e592c923db5b734249f3f659476c59210.png"
```

Comment:
@rishiagrawal2609 this wouldnt work because of how our nextjs app is configured. If we are using different site links we will have to include them in the config. 

Can you pull an updated PR by changing this image into an imgbb link please

Reply:
Resolved in latest commit